docs/community_isolation.rst

```diff
@@ -0,0 +1,82 @@
+This document assumes the reader has a basic understanding of running Gumby experiments, creating Dispersy communities and running them through the `TriblerExperimentScriptClient` class.
```

Comment:
http://stackoverflow.com/a/12365251 ` :code:`` ` should work, how was this handled in the rest of the docs?